lineage_test.go

```diff
@@ -32,9 +33,17 @@ func TestBindLineage(t *testing.T) {
 			t.Skip("case is tagged #slow, skipping for -short")
 		}
 
+		expected, ok := tc.Value("expectedErrorMessage")
```

Comment:
Do we plan to use `-- error --` file for such purpose? 🤔

Reply:
Initially I planned to assert on the type of error rather than on the message, so used a txtar tag `expectedErrorMessage`. But as I didn't figure out how to properly use `errors.Is` with cockroach errors, I rely on the error message for now and have been discussing with @spinillos that I should use `-- error --` file instead of the tag for that. This seems to be the pattern used across the other tests.

>The problem I see with that is that we reuse the same .txtar files for multiple use cases (different tests) and there might be different error outputs. Any idea/thought?

Good question. Need to think about it, don't have any good ideas right now.

Reply:
i've generally found the `-- error --` file to be a bit confusing. Like, i'm not sure what the standard use pattern is where you'd want to _not_ fail on an error. If it's an expected error, then it should be in the regular output for a particular test

in any case, i think our particular issues here will be solved by moving these txtar files into the `invalidlineages` testdata folder.